README.rst

```diff
@@ -120,16 +120,16 @@ The plugin itself holds a default config that can be used for any Coverity proje
         'stream': 'some_coverty_stream',
     }
 
-This default configuration build into the plugin, can be overriden through the conf.py of your project.
+This default configuration, built into the plugin, can be overridden through the *conf.py* of your project.
 
 
 -----
 Usage
 -----
 
-Inside your reStructured text file you can call a block `.. coverity-list:` which will generate the table
-with title and defined columns. For example to display CID, Classification, Action and Comment columns while
-filtering classification items with value `Bug` you should use the following snippet:
+Inside your reStructured text file, you can call a block `.. coverity-list:` which will generate the table
+with title and defined columns. For example, to display CID, Classification, Action and Comment columns while
```

Comment:
I would move comma after example to before while?
```
For example to display CID, Classification, Action and Comment columns, while filtering Classification items with value `Bug`, you should use the following snippet:
```

Reply:
I added a comma there and left the other comma, since a comma is used after introductory such as  _for example_ when they are followed by a series of items. I find it easier to read with the comma. 24ff2f805b9c3789c10252c0f528d871ab0face2